napari/benchmarks/benchmark_qt_viewer_labels.py

```diff
@@ -46,8 +37,8 @@ def setup(self):
         self.layer._last_cursor_coord = (511, 511)
         self.event = MouseEvent(
             type='mouse_move',
-            is_dragging=True,
-            pos=[500, 500],
+            press_event=MouseEvent(type='mouse_press', pos=(505, 500)),
```

Comment:
nice!

I *think* that to make the test equivalent, `pos` should be (511, 511) here? ie is `is_dragging` being interpreted by the callback using `layer._last_cursor_coord`?

Reply:
The `_last_cursor_coord` is only used in this benchmark. I will remove it.

Reply:
The value used in code is `_moving_coordinates`